app/views/admin_mailer/send_spam_alert.html.erb

```diff
@@ -0,0 +1,15 @@
+<% content_for :message do %>
+
+  <p>The following accounts have a suspicious level of traffic:</p>
+     <%@spam_score.sort_by{|k,v| -1*v}.map do |pseud,score| %>
+       <% user = User.find(pseud) %>
+       <h3>User <%= style_link( user.login , root_url+user_works_path(user))%> has a score of <%= score %></h3>
+       <ul>
+          <%@spam_works[pseud].each { |w| %>
+            <% work = Work.find(w) %>
+          <li><%=  style_link( work.title, work_url(work) ) %> <% if work.akismet_score %><%= ts('Flagged as spam') %><% end %></li>
+          <% } %>
+       </ul>
+     <% end %>
+
+<% end %>
```

Comment:
I've come to SarkenHound the frontend.

The indenting and spacing should be more like:

```
<% content_for :message do %>

  <p>The following accounts have a suspicious level of traffic:</p>
  <% @spam_score.sort_by{|k, v| -1 * v}.map do |pseud, score| %>
    <% user = User.find(pseud) %>
    <h3>User <%= style_link(user.login, root_url + user_works_path(user)) %> has a score of <%= score %></h3>
    <ul>
      <% @spam_works[pseud].each do |w| %>
        <% work = Work.find(w) %>
        <li><%= style_link(work.title, work_url(work)) %> <% if work.akismet_score %>(<%= ts('Flagged as spam') %>)<% end %></li>
      <% end %>
    </ul>
  <% end %>

<% end %>
```

I changed the `{}` to a `do...end`, and I've also put parentheses around "Flagged as spam" to match the txt version a bit more closely.

We should probably avoid introducing new HTML elements like `<h3>` into our HTML emails -- I guarantee someone's email client is going to make a mess. For now, let's stick with a simple `<%= style_link(user.login, root_url + user_works_path(user)) %> has a score of <%= score %>`, without any surrounding HTML tags. (I'm basing that on the batch subscription emails, which inexplicably don't use any HTML around the "so-and-so posted a new work:" bits, and which we haven't had any problems with. If it looks horrifically broken, `<p>` tags will be fine.)